graphbolt/src/shared_memory_utils.h

```diff
@@ -19,6 +19,30 @@
 namespace graphbolt {
 namespace sampling {
 
+/**
+ * @brief SharedMemoryHelper is a helper class to write/read data structure
+ * to/from shared memory.
+ */
+class SharedMemoryHelper {
+ public:
+  SharedMemoryHelper(const std::string& name, size_t size) : offset_(0) {}
```

Comment:
what size does it refer to?

Reply:
Should be the maximum size of the meta data. Will change the name later.